charts/newrelic-logging/templates/daemonset.yaml

```diff
@@ -118,8 +139,15 @@ spec:
           volumeMounts:
             - name: fluent-bit-config
               mountPath: /fluent-bit/etc
-            - name: var
-              mountPath: /var
+            - name: logs
+              mountPath: {{ .Values.fluentBit.linuxMountPath | default "/var" }}
+              {{- if and (.Values.fluentBit.persistence) (ne .Values.fluentBit.persistence.mode "hostPath") }}
+              readOnly: true
+              {{- end }}
+            {{- if and (.Values.fluentBit.persistence) (eq .Values.fluentBit.persistence.mode "persistentVolume") }}
+            - name: fb-db-pvc
+              mountPath: /db
+            {{- end }}
```

Comment:
This is how the DB path will be observed from _inside_ the pod (and this will be a new pod). Even if we change it here, we won't incur any breaking change and we will have a clear separation of the "logs" mount and the "db" mount:

```
{{- if ne .Values.fluentBit.persistence.mode "none"}}
  - name: fb-db
    mountPath: /db
{{- end }}
```

Reply:
It'll break for people that already have their values with the db path set to `/var/log/flb_kube.db`